k-distribution/tests/regression-new/spec-rule-application/def02-spec.k.out

```diff
@@ -1 +1,8 @@
-#Top
```

Comment:
@radumereuta Changing the def02-spec.k resulted in a different test output. Is this the correct approach??